app/helpers/browse_helper.rb

```diff
@@ -1,28 +1,14 @@
 module BrowseHelper
-  ACTION_LINK_DATA = {
-    benefits: [
-      { lang: "browse.check_benefits_and_financial_support", href: "/check-benefits-financial-support" },
-    ],
-    business: [
-      { lang: "browse.hmrc_online_services", href: "/log-in-register-hmrc-online-services" },
-      { lang: "browse.self_assessment_tax_returns", href: "/self-assessment-tax-returns" },
-      { lang: "browse.pay_employers_paye", href: "/pay-paye-tax" },
-    ],
-  }.freeze
-
-  def display_action_links_for_slug?(slug)
-    ACTION_LINK_DATA.key?(slug.to_sym)
+  def display_popular_links_for_slug?(slug)
+    I18n.exists?(slug.to_s, scope: "browse.popular_links")
   end
 
-  def action_link_data(slug)
-    links = ACTION_LINK_DATA[slug.to_sym]
-    return [] unless links
-
-    links.each_with_index do |link, index|
-      link[:text] = I18n.t(link[:lang])
-      link[:ga4_text] = I18n.t(link[:lang], locale: :en)
-      link[:index_link] = index + 1
-      link[:index_total] = links.length
+  def each_popular_link_for_slug(slug)
+    links = I18n.t(slug.to_s, scope: "browse.popular_links")
+    count = links.length
+    links.each.with_index(1) do |link, index|
+      analytics_label = I18n.t(link[:analytics_label_key], scope: "browse.analytics_labels", locale: :en)
+      yield link, analytics_label, index, count
```

Comment:
It looks like the action link component accepts a `data_attributes` option (docs [here](https://components.publishing.service.gov.uk/component-guide/action_link))

So I think it might be clearer to just pass a single `data_attributes` key to the view that contains all the values,  rather than having elements of the data attributes hash defined in different files. Something like:

```
def popular_link_for_slug(slug)
  links = I18n.t(slug.to_s, scope: "browse.popular_links")
  count = links.length
  links.map.with_index(1) do |link, index|
    {
      text: link[:title],
      href: link[:url],
      data_attributes: {
       module: "ga4-link-tracker",
       ga4_track_links_only: "",
       ga4_link: {
          event_name: "navigation",
          type: "action",
          index_link: index,
          index_total: count,
          text: link[:title],
        },
      }
    }
  end
end
```

What do you think? Similar to [this](https://github.com/alphagov/collections/blob/main/app/presenters/taxon_organisations_presenter.rb#L62-L83)

Reply:
The requirement of the ticket was:

`Move the[ data attributes](https://github.com/alphagov/collections/blob/main/app/views/browse/show.html.erb#L34-L51) for tracking into the view`

Has the requirement changed?

Reply:
The intention of the requirement was to have all the data attributes defined in one place. Rather than some being defined in the helper, and some being defined in the view.

 But as there's a pattern already used in this app of passing through the `data_attributes` key I think this is a nicer way to meet the objective of all in one place.

Reply:
I'll also remove the partial, as it's no longer necessary, as it only contains one line and is used in one place only.

Reply:
Changed.